client/active-game/active-game-manager.js

```diff
@@ -274,6 +275,15 @@ async function doLaunch(gameId, serverPort, settings) {
     throw new Error(`StarCraft path ${starcraftPath} not valid: ` + JSON.stringify(checkResult))
   }
   const isRemastered = checkResult.remastered
+  if (isRemastered) {
+    try {
+      // Blizzard reinitializes their settings file everytime the SC:R is opened through their
+      // launcher. So we must do the same thing with our own version of settings before each game.
+      await ipcRenderer.invoke(SCR_SETTINGS_OVERWRITE)
+    } catch (err) {
+      throw new Error('Error overwriting SC:R settings: ' + err)
```

Comment:
Feels like this should probably just throw the original error instead of discarding it in place of a toString'd version of it? (e.g. just remove the `try/catch` here and do the `await` right inside the if)